compiler/parser/python.lalrpop

```diff
@@ -646,15 +647,16 @@ KwargParameter<ArgType>: Option<Box<ast::Arg>> = {
 };
 
 ClassDef: ast::Stmt = {
-    <decorator_list:Decorator*> <location:@L> "class" <name:Identifier> <a:("(" ArgumentList ")")?> ":" <body:Suite> <end_location:@R> => {
+    <decorator_list:Decorator*> <location:@L> "class" <name:Identifier> <a:("(" ArgumentList ")")?> ":" <body:Suite> => {
```

Comment:
Just modified `ClassDef` and `FuncDef` as a test.

Reply:
@DimitrisJim Thanks for the comment. Yes, I need to apply the same change to other compound statements. I'm working on that now.

Reply:
ah yes, just saw it's still a draft! :smile:

Reply:
@DimitrisJim Updated other compound statements.